Aspen-Discovery/aspen-discovery
# DIS-43: Standarize parameters used for Solr indexing

Calls to SolrClient commit() use many permutations of the boolean parameters for waitFlush, waitSearcher, and softCommit. The calls are uncommented so the justification for the settings is unknown. Based on recommendations from a Solr expert we should set all these calls to commit should use false,false,true

Test Plan:
1) Apply this patch
2) Run actions that should trigger chnages to solr indexes
3) No changes in search results should be noted
